addons/website/static/src/snippets/s_image_gallery/000.js

```diff
@@ -16,6 +19,12 @@ const GalleryWidget = publicWidget.Widget.extend({
     start() {
         this._super(...arguments);
         this.originalSources = [...this.el.querySelectorAll("img")].map(img => img.getAttribute("src"));
+
+        this._maybeEnableExpandControl();
```

Comment:
Why we need this here?

Reply:
This function checks whether the expandable setting is enabled for the snippet. If it is, it displays the `show-more` button, allowing users to reveal the hidden gallery item.

Reply:
Yes! we can avoid _maybeEnableExpandControl fn in 000.js
This fn was mainly showing the `show-more` button if the expandable setting was enabled.

Your suggestion works and we can shift this logic to destroy of `options.js`.
So, when the snippet is saved we can check if expandable setting is ON and unhide the `show-more` button avoiding that extra fn in 000.js.